frontend/src/container/GuestUpload.jsx

```diff
@@ -137,16 +141,16 @@ class GuestUpload extends React.Component {
     }
 
     this.setState(previousState => ({
-      fileToUploadList: [
-        ...previousState.fileToUploadList,
-        ...newFileList.filter(newFile => !state.fileToUploadList.some(stateFile => stateFile.name === newFile.name))
+      fileUploadList: [
+        ...previousState.fileUploadList,
+        ...newFileUploadList
```

Comment:
Does this account for duplicates?

Reply:
The code above takes care of removing the duplicates (and warns the user if there is one).

Reply:
Ah, right, so maybe don't change the state if  `alreadyPresentList.length`